fmlloader/src/main/java/net/minecraftforge/fml/loading/moddiscovery/ModValidator.java

```diff
@@ -106,7 +106,8 @@ public ITransformationService.Resource getModResources() {
 
     private List<EarlyLoadingException.ExceptionData> validateLanguages() {
         List<EarlyLoadingException.ExceptionData> errorData = new ArrayList<>();
-        for (Iterator<ModFile> iterator = this.candidateMods.iterator(); iterator.hasNext(); ) {
+        Iterator<ModFile> iterator = this.candidateMods.iterator();
+        while (iterator.hasNext()) {
```

Comment:
Should be literally the same compiled code. I personally like using the former pattern for itterators. Little jumbled but its nice.

Reply:
Yes please.
And ya, the scope of the itr variable would be slightly different, One being the end of the for loop the other being the end of the function.